gdal/src/main/scala/geotrellis/contrib/vlm/gdal/GDALDataPath.scala

```diff
@@ -0,0 +1,30 @@
+package geotrellis.contrib.vlm.gdal
```

Comment:
License headers

Reply:
For the source code use `createHeaders` and `test:createHeaders` to generate headers in tests.